ledger/test-common/BUILD.bazel

```diff
@@ -14,24 +14,28 @@ load(
     "//language-support/scala/codegen:codegen.bzl",
     "dar_to_scala",
 )
-load(
-    "//daml-lf/language:daml-lf.bzl",
-    "lf_dev_version",
-    "lf_latest_version",
-    "lf_preview_version",
-    "lf_stable_version",
-)
 load("@scala_version//:index.bzl", "scala_major_version")
+load("@scala_version//:index.bzl", "scala_major_version")
+load(":test-common.bzl", "test_common_configurations", "test_common_lf_targets")
 
-# Tuples of (lf_version, name_suffix)
-# The DAML files are compiled once for each daml-lf version listed here.
-test_common_configurations = [
-    (lf_stable_version, ""),
-    (lf_latest_version, "-latest"),
-    (lf_dev_version, "-dev"),
-] + [
-    (v, "-preview")
-    for v in lf_preview_version
+[
+    [
+        alias(
+            name = "%s-%s%s" % (target_prefix, name, target_suffix),
```

Comment:
it’s hard to see what is actually being aliased here quickly. I think a comment for future readers would be helpful with an example.